### ISSUE TITLE
Authentication tutorial is outdated

### ISSUE THREAD

**Issue by ze-phyr-us** [tweepy/tweepy#521]
1. The attributes `auth.request_token.key` and `auth.request_token.secret` do not exist anymore. `auth.request_token` is now a dictionary and the key and secret are found in `auth.request_token['oauth_token']` and `auth.request_token['oauth_token_secret']`.
2. `auth.set_request_token` no longer exists and it seems one has to assign the above dictionary directly to `auth.request_token`.
3. `auth.access_token.key, auth.access_token.secret` don't exist anymore. One has to use `auth.access_token` and `auth.access_token_secret`.

Did not find any mentions of these breaking changes in the docs or the changelog.

**Comment by marius92mc** [comment]
I subscribe to the above reply. The tutorial really needs to be updated.

**Comment by TristanH** [comment]
+1 Also ran into the second one listed

**Comment by gserapio** [comment]
Does anyone have more details on the second problem? Would the code become `auth.request_token = token` ?

**Comment by qmcree** [comment]
Also ran into issues with these changes after upgrading. Since retrieving the request token keypair, directing the user to the authorization URL, and getting the access token usually all happen in separate threads (retrieving the request token keypair from a temporary store), I'm not sure why the `set_request_token` method was removed. Here's the update:

```
auth = tweepy.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
auth.request_token = dict(oauth_token=REQUEST_TOKEN, oauth_token_secret=REQUEST_TOKEN_SECRET)
return auth.get_access_token(VERIFIER)
```

**Comment by Harmon758** [comment]
This was resolved with https://github.com/tweepy/tweepy/commit/ac5eb8019e03fe1185b26ddbdbf3b5f44d9a4486 (#526) and was later fixed by https://github.com/tweepy/tweepy/commit/6385c3bf51b7bfce332450027d8832cf2cb0f6d7 (#799) and https://github.com/tweepy/tweepy/commit/f381a042b8eeab14646e791efdf85399758f67c3 (#1146).
The first update was released with v3.2.0, the first fix with v3.6.0, and the latter fix is now on [the latest version of the documentation](https://tweepy.readthedocs.io/en/latest/auth_tutorial.html).
